src/representations/cutoff_functions.hh

```diff
@@ -123,6 +123,10 @@ namespace rascal {
      *        |
      *        ╰ c / (c + (r/r_0)^m), else
      *
+     * c -> rate
+     * r_0 -> scale
+     * m -> exponent
```

Comment:
Good. Perhaps we should add a comment to the effect that `c` is almost always left at 1?